Objects/src/RecoLepton.cc

```diff
@@ -39,6 +41,8 @@ RecoLepton::RecoLepton(const ChargedParticle & lepton,
   , jetIdx_(jetIdx)
   , genPartFlav_(genPartFlav)
   , genPartIdx_(genPartIdx)
+  , isBoostedLepton_(isBoosted)
+  , lsf3_(lsf3)
```

Comment:
Hardcode the default values (`false` and `-1`). There is no particular need to provide those as arguments to the constructor.